libs/category/src/actions/category-page.actions.ts

```diff
@@ -16,11 +16,11 @@ export enum DaffCategoryPageActionTypes {
   CategoryPageLoadAction = '[Daff-Category] Category Page Load Action',
   CategoryPageLoadSuccessAction = '[Daff-Category] Category Page Load Success Action',
   CategoryPageLoadFailureAction = '[Daff-Category] Category Page Load Failure Action',
-  ChangeCategoryPageSizeAction = '[Daff-Category] Change Category Page Size Action',
-  ChangeCategoryCurrentPageAction = '[Daff-Category] Change Category Current Page Action',
-  ChangeCategorySortingOptionAction = '[Daff-Category] Change Category Sorting Option Action',
-  ChangeCategoryFiltersAction = '[Daff-Category] Change Category Filters Action',
-  ToggleCategoryFilterAction = '[Daff-Category] Toggle Category Filter Action'
+  CategoryPageChangeSizeAction = '[Daff-Category] Change Category Page Size Action',
```

Comment:
I think maybe this should be `[Daff Category Page] Change Page Size Action`, because the convention isn't that we put the package name inside the [ ].

Reply:
I'm not sure we have a convention for the `[]` These actions are somewhat becoming overly verbose.

```
[@daffodill/category] Category Page Load Action
```

seems ok to me.

```
[@daffodill/category] Page Load Action 
```

also seems good

```
[Daffodil Category Page] Load Action 
```

Seems less good, but I'm not really against it.

Reply:
On the one hand, I like the idea of having the package name inside the [ ], but on the other, I think that will lead to longer action type names. For example:

```
'[Daffodil Configurable Product] Toggle Attribute Action'
vs 
'[@daffodil/product] Configurable Product Toggle Attribute Action'
```
and 
```
'[Daffodil Product] Load Action'
vs
'[@daffodil/product] Product Load Action'
```
or even
```
'[Daffodil Category Page] Toggle Filter Action'
vs
'[@daffodil/category] Category Page Toggle Filter Action'
```

But I don't really mind the longer names to be honest, so I'd be down for `[packageName] Verbose Action Name` if that's good with you